src/search-scenario/azext_search_scenario/_clierror.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.azclierror import ClientRequestError
+
+# pylint: disable=unnecessary-pass
+
+
+class ScenarioSearchError(ClientRequestError):
```

Comment:
Seems the naming is not consistent between `ScenarioSearchError` and `SearchScenarioCommandsLoader`. (ScenarioSearch vs SearchScenario)

Reply:
Done